README.md

```diff
@@ -75,7 +78,7 @@ Reports whether user has withdrawn consent to sell their data:
 
 ```js
 {
-    doNotSell: Boolean
+    doNotSell: Boolean;
```

Comment:
the example above uses js-style `,` – we should probably match them. want to go with TS version?

Reply:
The linter added this. But I supposed if it is a type definition TS makes more sense.